gravitee-apim-console-webui/src/components/alerts/alertEnginePromotion/alert-engine-promotion-component.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const AlertEnginePromotionComponent: ng.IComponentOptions = {
+  bindings: {},
+  template: require('./alert-engine-promotion.html'),
+  /* @ngInject */
+  controller: function () {
+    this.contactAlerts = 'https://www.gravitee.io/contact-us-alerts'
+    this.gotoLearn = () => {
+      console.log("learning content");
```

Comment:
Can't we have the expected action linked to this button yet ?

Reply:
not yet, but will change John Green

Reply:
Button link added. The links are available in the figma, fyi.